doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -242,6 +244,118 @@ Learn how to write filters in this [filter pattern syntax doc](https://docs.aws.
 [KubDoc]: https://kubernetes.io/docs/concepts/cluster-administration/logging/
 [ConfigLink]: https://raw.githubusercontent.com/ray-project/ray/releases/2.4.0/doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml
 
+### Method 2: Grafana Loki
```

Comment:
Would you mind creating a new document specifically for persisting KubeRay operator logs and leaving this document solely for persisting RayCluster logs?

Maybe we can rename this doc from "Log Persistence" to "Persist KubeRay logs", and use "Persist KubeRay operator logs" for the new doc.

Reply:
Sure, should I create another PR to complete this, or should I include it in the current PR?

Reply:
Update: I changed "Persist KubeRay logs" to "Persist Ray logs" for better clarity.

Reply:
> I will create another PR to add Persist KubeRay Operator Logs doc. This PR will focus on persist KubeRay logs.

I am a bit confused. This section looks like persisting KubeRay operator logs.